LiskHQ/lisk-sdk
# Use consistent naming format for key in key-value table - Closes #5064

### What was the problem?

This PR resolves #5064 

### How was it solved?

- Updated key value to format `modulename:keyName`

### How was it tested?

- Update tests 
- Run application and check database with key
